Artifacts/windows-CreateDomain/setupDomain.ps1

```diff
@@ -15,7 +15,7 @@ if ((gwmi win32_computersystem).partofdomain -eq $false)
   Install-ADDSForest -CreateDnsDelegation:$false `
     -DatabasePath "C:\Windows\NTDS" `
     -DomainMode "Win2012R2" `
```

Comment:
That's the functional Level of the AD Domain. This does also work on Server2016.